### ISSUE TITLE
UI lag with many conversations

### ISSUE THREAD

**Issue by kousu** [siacs/Conversations#1889]
Conversations seems to lag hard when many conversations are open. This is more noticeable on slower devices (which are the kind I tend to buy).

Ideally I would like the number of UI widgets to not affect the scrolling speed. If that's not possible, as a workaround, can there be a "close all conversations but not MUCs" button?

#### General information

* **Version:** eg 1.12.3
* **Device:** eg Polaroid Link A4
* **Android Version:** eg Android 4.4.4
* **Conversations source:** F-Droid

#### Steps to reproduce

1. Open a LOT of conversations
2. Try to scroll the open conversations list -> slow as molasses!
3. Remove most of the conversations -> fast again!

**Comment by iNPUTmice** [comment]
What's a lot?

**Comment by iNPUTmice** [comment]
Also you might want to update to >=1.12.5 might just be that you were effect by the long message bug that was fixed in 1.12.5

**Comment by kousu** [comment]
You know what, I can't reproduce it now.

I tried to create 30 conversations, and while it does lag out sometimes with that, I think that's just that the phone is slow because it lags in the same ways for about the same amount of time with only 5. It doesn't seem to increase with more conversations after all. I thought maybe having the little picture previews in the conversation list when the last message was a pic might be the culprit so I made 5 of those which is more than I had before I closed a bunch of conversations, but none of that is making it any laggier than it is with just a few.

Mostly I was just wondering if there's an obvious reason why there'd be a slowdown with more conversations open. Would it be something about the Android GUI API? I know Java Swing apps get laggy when they need to make lots of widgets, even if not all the widgets are on screen simultaneously.

Did the long message bug cause lag? And closing the conversations would clear the long messages from RAM?
